dockerfile/parser/parser.go

```diff
@@ -85,6 +110,7 @@ var (
 	tokenWhitespace      = sRegexp.Delayed(`[\t\v\f\r ]+`)
 	tokenEscapeCommand   = sRegexp.Delayed(`^#[ \t]*escape[ \t]*=[ \t]*(?P<escapechar>.).*$`)
 	tokenPlatformCommand = sRegexp.Delayed(`^#[ \t]*platform[ \t]*=[ \t]*(?P<platform>.*)$`)
+	reHeredoc            = regexp.MustCompile(`^(\d*)<<(-?)([^<]*)$`)
```

Comment:
Any reason this is using `regexp.MustCompile()` rather than `sRegexp.Delayed()` like the rest?

Reply:
This variable is not needed. I have removed it.